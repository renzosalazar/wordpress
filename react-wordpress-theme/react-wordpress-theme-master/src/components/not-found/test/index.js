/**
 * Test for NotFound component.
 *
 * @package
 */

/**
 * External dependencies
 */
import renderer from 'react-test-renderer';

/**
 * Internal dependencies
 */
import NotFound from '../index';

describe( 'Test NotFound Component', () => {
	it( 'Should match NotFound snapshot and render correctly', () => {
		const component = renderer.create( <NotFound /> );

		const tree = component.toJSON();

		expect( tree ).toMatchSnapshot();
	} );
} );
