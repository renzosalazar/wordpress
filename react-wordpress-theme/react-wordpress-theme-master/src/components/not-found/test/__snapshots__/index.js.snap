// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test NotFound Component Should match NotFound snapshot and render correctly 1`] = `
<div>
  
  <div
    className="content"
  >
    <h3>
      404
    </h3>
    Sorry, nothing here.
  </div>
  
</div>
`;
